code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -377,6 +386,14 @@
 							feedback_add_details("wizard_spell_learned","WM") //please do not change the abbreviation to keep data processing consistent. Add a unique id to any new spells
 							add_spell(new/spell/aoe_turf/conjure/pontiac,H)
 							temp = "This spell summons a glorious, flaming chariot that can move in space and through walls.  It also has an extremely long cooldown."
+						if("bundle")
+							feedback_add_details("wizard_spell_learned","SB")
+							new /obj/item/weapon/storage/box/spellbook(get_turf(H))
+							temp = "You have purchased the spellbook bundle."
+							uses -= 4
```

Comment:
Can you give a quick justification to how this system works, that it does uses -= 4 and max_uses -= 5, for me?